domains/bernardofoegbu.json

```diff
@@ -6,6 +6,9 @@
     "email": "obinnaofoegbu71@gmail.com"
   },
   "record": {
-    "CNAME": "o-bernardofoegbu.github.io"
+    "CNAME": "o-bernardofoegbu.github.io",
```

Comment:
You can not use CNAME rexords together with any other record

Reply:
okay... will use just the txt